.eslintrc.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
     'no-console': 0,
 
     'import/no-named-as-default': 0,
+    'import/no-named-as-default-member': 0,
```

Comment:
Seems to build ok without it, so maybe not.  I needed it at one point, and we have that rule in pvw.  But I'll get rid of it.